content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,17 @@
+## Bug fixes and minor enhancements
+
+- Add a precise SHA reference to the release manifest ([Pull Request 14959](https://github.com/istio/istio/pull/14959))
```

Comment:
I don't think this needs to be user facing

Reply:
```suggestion
```